addons/account_peppol/models/res_config_settings.py

```diff
@@ -97,6 +80,14 @@ def _compute_account_peppol_endpoint_warning(self):
                 config.account_peppol_endpoint_warning = _("The endpoint number might not be correct. "
                                                            "Please check if you entered the right identification number.")
 
+    @api.depends('company_id.is_account_peppol_participant')
+    def _compute_is_any_account_peppol_participant(self):
+        peppol_participants_sudo = self.env['res.company'].sudo().search([
+            ('is_account_peppol_participant', '=', 'True')
+        ])
+        for config in self:
+            config.is_any_account_peppol_participant = len(peppol_participants_sudo) > 0
```

Comment:
I don't get it. The settings are only there for a single company. I don't understand why you need something cross-company here

Reply:
@dylankiss :scream: ok make sense